packages/loot-core/src/server/spreadsheet/tests/graph.ts

```diff
@@ -3,12 +3,12 @@ import assert from 'assert';
 
 // If using from the NPM package, this line would be
 // import Graph from "graph-data-structure";
-import Graph from '../data-compute/graph-data-structure';
+import Graph from '../graph-data-structure';
```

Comment:
This test file can be cleaned up entirely, I think none of the tests actually work and they are not run by the tester.